File: 1480. Running Sum of 1d Array/main.cpp
#include <vector>
#include <iostream>


class Solution {
public:
    std::vector<int> runningSum(std::vector<int>& nums) {
        for(unsigned int i = 1; i < nums.size(); i++){
            nums[i] += nums[i-1];
        }
        return nums;
    }
};


int main(int argc, char const *argv[])
{
    std::vector inp = {1,2,3,4};
    Solution s = Solution();
    std::vector<int> res = s.runningSum(inp);

    for(unsigned int i = 0; i < res.size(); i++){
        std::cout << res[i] << ", ";
    }
    return 0;
}

File: 1480. Running Sum of 1d Array/main.py
class Solution(object):
    def runningSum(self, nums: list[int]) -> list[int]:
        out = []
        sum = 0
        for i in range(len(nums)):
            sum += nums[i]
            out.append(sum)
        return out



def main():
    inp = [1,2,3,4]
    print(Solution.runningSum(Solution(), inp))


if __name__ == "__main__":
    main()